testsuite/features/init_clients/sle_minion.feature

```diff
@@ -41,12 +41,12 @@ Feature: Bootstrap a Salt minion via the GUI
     And I follow "Proxy" in the content area
     Then I should see "sle_minion" hostname
 
-  Scenario: Migrate this minion to SLE 15 SP3
+  Scenario: Migrate this minion to SLE 15 SP4
     Given I am on the Systems overview page of this "sle_minion"
     When I follow "Software" in the content area
     And I follow "Product Migration" in the content area
     And I wait until I see "Target Products:" text, refreshing the page
-    And I wait until I see "SUSE Linux Enterprise Server 15 SP3 x86_64" text
+    And I wait until I see "SUSE Linux Enterprise Server 15 SP4 x86_64 (BETA)" text
```

Comment:
:thinking: we need to pay attention to this because the beta tag will be gone eventually right? isn't it enough without the beta, as a partial match?

Reply:
Good catch. I thought I removed everything SP4 BETA related.

Reply:
Beware you might need to select elements with a BETA in the name. If our checkbox selection steps and our dropdown selection steps work with partial text, all fine. If not, it will break.

Maybe not in this feature, though. I'm thinking more particularly at the setup wizard.

Just thinking aloud.